examples/all-clusters-app/esp32/main/main.cpp

```diff
@@ -85,8 +85,9 @@ namespace {
 class AppCallbacks : public AppDelegate
 {
 public:
+    void OnCommissioningSessionEstablishmentStarted() {}
     void OnCommissioningSessionStarted() override { bluetoothLED.Set(true); }
-    void OnCommissioningSessionStopped() override
+    void OnCommissioningSessionStopped(CHIP_ERROR err) override
```

Comment:
This changes public API. This should have been a new callback for status in addition to OnCommissioningSessionStopped.

Furthermore, usage of the `err` argument is not documented.

Reply:
Addressing part of this in this PR:  [#28138](https://github.com/project-chip/connectedhomeip/pull/28138)

and tracking the rest in this issue: [#28136](https://github.com/project-chip/connectedhomeip/issues/28136)